roles/contiv_network/templates/netplugin.j2

```diff
@@ -1 +1 @@
-NETPLUGIN_ARGS='-plugin-mode {{netplugin_mode}} -vlan-if {{netplugin_if}} -vtep-ip {{node_addr}}'
+NETPLUGIN_ARGS='-plugin-mode {{netplugin_mode}} -vlan-if {{netplugin_if}} -vtep-ip {{node_addr}} -fwd-mode {{fwd_mode}}'
```

Comment:
@mapuri yes thats right . Though the binary pulled https://github.com/contiv/ansible/blob/master/roles/contiv_network/tasks/main.yml#L23  would support this flag its more intended for the new set of netplugin/ofnet changes made. I will bump up the release number once a binary generated with the changes in netplugin and ofnet.